### ISSUE TITLE
On Red Hat family, the value of the platform_family attribute is "rhel", not "redhat"

### ISSUE THREAD

**Issue by abe4tawa8** [treasure-data/chef-td-agent#73]
On Red Hat family, `node["platform_family"]` is `rhel`, not `redhat`.

https://github.com/treasure-data/chef-td-agent/blob/485bd214d113d360feaf4387d39fc50552199037/libraries/version.rb#L13-L28
```ruby
    def reload_available?
      case node["platform_family"]
      when "debian"
        # td-agent's init script for debian starts supporting reload starting from 2.1.4
        if node["td_agent"]["version"]
          ::Gem::Version.new("2.1.4") <= ::Gem::Version.new(node["td_agent"]["version"].to_s)
        else
          false
        end
      when "redhat"
        # td-agent's init script for redhat is supporting reload
        true
      else
        false
      end
    end
```

https://github.com/chef/ohai/blob/master/lib/ohai/plugins/linux/platform.rb#L182-L183
```ruby
    when /oracle/, /centos/, /redhat/, /scientific/, /enterpriseenterprise/, /amazon/, /xenserver/, /cloudlinux/, /ibm_powerkvm/, /parallels/, /nexus_centos/ # Note that 'enterpriseenterprise' is oracle's LSB "distributor ID"
      platform_family "rhel"
```

**Comment by yyuu** [comment]
Uggggggh. Good catch :+1: I'll fix it soon....

**Comment by abe4tawa8** [comment]
Thank you for fixing. :smile:
I want to use the new version 2.1.5.
But it is not available on supermarket.chef.io.
When is it available?

**Comment by yyuu** [comment]
I released it as 2.1.4 on Supermarket.

**Comment by abe4tawa8** [comment]
v2.1.5 f4ae0f7ce48e1cd59d75d8226bfa79e6c7307590 is a next version, I understand.
Thank you.
